### ISSUE TITLE
genambig.to.genbinary fails when alleles are letters instead of numbers

### ISSUE THREAD

**Issue by jkibilds** [lvclark/polysat#4]
Hello!

I have a GeneMapper genotype table of SNP data (the alleles are ATGC not numbers like in the case of microsatellites). I have imported it to polysat as a `genambig` object. My goal is to get in data from GeneMapper and to export it as a `genind` object for futher work in adegenet R package. This is my code and I attach [one of my GeneMapper files](https://github.com/lvclark/polysat/files/5202125/my_GM_file.txt)

```
genotypes <- read.GeneMapper(infiles = "my_GM_file.txt", forceInteger = FALSE)
genotypes <- reformatPloidies(genotypes, output="one")
Ploidies(genotypes) <- 2
mydata <- gendata.to.genind(genotypes)
```
And this is the error message that I get instead of a `genind` object:
```
Error in `Genotypes<-`(`*tmp*`, value = domdata) : 
  Column names should be of form "locus.allele".
```
I tried to investigate the problem and it seems that the error is caused by `genambig.to.genbinary` function that is called by `gendata.to.genind`. At least the same error occurs when I separately try to use `genambig.to.genbinary` to convert my `genambig` object to `genbinary`.
I also verified that these commands work if I use GeneMapper example files from this repository as input. Further, I replaced the numeric alleles in [GeneMapperCBA28.txt](https://github.com/lvclark/polysat/blob/master/inst/extdata/GeneMapperCBA28.txt) with ATGC (produced the same error) and replaced ATGC with numeric alleles in my files (worked without an error). So I conclude that having letters for alleles is likely the problem. But that is as far as I can get with my R skills. Can you please look into this and provide a solution?

I am using polysat 1.7-4 and adegenet 2.1.3

Thanks!

**Comment by lvclark** [comment]
Thanks for bringing this to my attention!  Not a lot of people use polysat with SNP data, so there was a bug in `read.GeneMapper` that I wasn't aware of.  I have fixed it in version 1.7-5 on GitHub.  Just run

``` r
devtools::install_github("lvclark/polysat")
```

Keep in mind the `genind` object will still be presence/absence format rather than codominant format.

**Comment by jkibilds** [comment]
Now it works well. Thank you very much!
